scripts/globals/bcnm.lua

```diff
@@ -967,13 +967,15 @@ function EventFinishBCNM(player, csid, option)
             local stat = player:getStatusEffect(dsp.effect.BATTLEFIELD)
             local bfid = stat:getPower()
             local item = getItemById(player, bfid)
+            local _, initiatorName = player:getBattlefield():getInitiator()
```

Comment:
you really don't want a variable named `_` do you? ..what is that being used for?

Reply:
Zyn suggested `initiatorId` instead of `_` I'd go with that if you don't have any better ideas. Descriptive vars are better than throwaways. But I didn't see where its being used.

Reply:
The whole point of the _ var in lua is that it is not stored. It is best practice with lua. I don't ever use it, it is literally a throw away.

Reply:
Will change if you feel it is 100% necessary.

However, this is standard practice for things in Lua and even Python when you are dealing with  multiple returns and you don't want them.

Reply:
didn't realize it was returning 2 things. name it what it is in case we use it later.

Reply:
Done